frontends/dashboard/src/components/Sidebar.tsx

```diff
@@ -73,31 +91,59 @@ export const Sidebar = () => {
           <div class="border-b px-4 py-3">
             <h5 class="font-semibold text-neutral-600">Admin Tools</h5>
             <div class="flex flex-col items-start space-y-1 py-2">
-              <div class="flex items-center text-neutral-800 hover:text-fuchsia-800">
-                <a
-                  href={`${searchUiURL}${orgDatasetParams()}`}
-                  target="_blank"
-                  class="flex items-center"
+              <div class="flex flex-col items-center text-neutral-800">
+                <button
+                  onClick={() => setShowSubMenu(!showSubMenu())}
+                  class="flex items-center gap-2 hover:text-fuchsia-800"
                 >
-                  Search playground{" "}
+                  {showSubMenu() ? <FaSolidAngleDown /> : <FaSolidAngleRight />}
+                  <span>Search playground</span>
                   <IoOpenOutline class="ml-1 inline-block h-4 w-4" />
-                </a>
+                </button>
+                <Show when={showSubMenu()}>
+                  <div class="space-y-1 pb-1 pl-4 pt-1">
+                    <a
+                      href={`${searchUiURL}/group${orgDatasetParams()}`}
+                      target="_blank"
+                      class="flex items-center hover:text-fuchsia-800"
+                    >
+                      Group Data{" "}
+                      <IoOpenOutline class="ml-1 inline-block h-4 w-4" />
+                    </a>
+                    <a
+                      href={`${searchUiURL}/create${orgDatasetParams()}`}
+                      target="_blank"
+                      class="flex items-center hover:text-fuchsia-800"
+                    >
+                      Create Data{" "}
+                      <IoOpenOutline class="ml-1 inline-block h-4 w-4" />
+                    </a>
+                    <a
+                      href={`${searchUiURL}/upload${orgDatasetParams()}`}
+                      target="_blank"
+                      class="flex items-center hover:text-fuchsia-800"
+                    >
+                      Upload Data{" "}
+                      <IoOpenOutline class="ml-1 inline-block h-4 w-4" />
+                    </a>
```

Comment:
Change this to: 

- View Groups
- Create Chunk
- Upload File

Reply:
fixed